src/components/application-loader/initializers/i18n.ts

```diff
@@ -19,7 +19,7 @@ export const setUpI18n = async (frontendAssetsUrl: string): Promise<void> => {
       fallbackLng: 'en',
       debug: process.env.NODE_ENV !== 'production',
       backend: {
-        loadPath: `${ frontendAssetsUrl }locales/{{lng}}.json`
+        loadPath: `${frontendAssetsUrl}locales/{{lng}}.json`
```

Comment:
This seems to be wrong. IIRC we WANT these spaces

Reply:
That's right. I still have to find an option for that though.

Reply:
Apparently there is no option for this and https://github.com/prettier/prettier/issues/95 suggests prettier won't change this.